lib/api/traversing.js

```diff
@@ -36,13 +29,10 @@ var select = require('css-select'),
  * @see {@link http://api.jquery.com/find/}
  */
 exports.find = function (selectorOrHaystack) {
-  var elems = _.reduce(
-    this,
-    function (memo, elem) {
-      return memo.concat(_.filter(elem.children, isTag));
-    },
-    []
-  );
+  var elems = [];
+  for (const elem of Array.from(this)) {
```

Comment:
`this` is already an iterable.

```suggestion
  for (const elem of this) {
```